src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
```

Comment:
I almost wonder if there should be two list components since we'll be managing to separate lists of dependents theoretically.

Reply:
yeah, so I made a separate ticket to figure out the logic for the list component and that could very well include a refactor that breaks these out into separate components.